oci/oci.go

```diff
@@ -604,6 +604,97 @@ func waitContainerStop(ctx context.Context, c *Container, timeout time.Duration)
 	return nil
 }
 
+// WaitContainerStateStopped waits for the container in-memory status to be
+// updated to 'stopped'.
+func WaitContainerStateStopped(ctx context.Context, c *Container, timeout int64) error {
+	// No need to go further and spawn the go routine if the container
+	// is already in the expected status.
+	if c.state.Status == ContainerStateStopped {
+		return nil
+	}
+
+	done := make(chan struct{})
+	// we could potentially re-use "done" channel to exit the loop on timeout
+	// but we use another channel "chControl" so that we won't never incur in the
```

Comment:
This comment is copy/pasting wording which was updated in parallel in #1335.  We can handle it a few ways:

a. Replace it with a copy/paste of the new wording (easy for readers, but not very DRY).
b. Replace it with a comment like “Follow the waitContainerStop approach with chControl”, and expect readers to scroll up to read the details (tedious for readers, brittle but DRY).
c. Drop the comment here, moving wording like (b) into the commit message (DRY, easy for readers who use `git blame` to understand existing code, hard for other readers).

My preference is (c), but all of them would work.